Library/Formula/vdirsyncer.rb

```diff
@@ -0,0 +1,91 @@
+class Vdirsyncer < Formula
+  desc "Synchronize calendars and contacts"
+  homepage "https://github.com/untitaker/vdirsyncer"
+  url "https://pypi.python.org/packages/source/v/vdirsyncer/vdirsyncer-0.6.0.tar.gz"
+  sha256 "0d2a9677b086cfbe5fe5e7cb5e55db4c1afab62cb5dc56aeaff6e95d34bf60d5"
+  head "https://github.com/untitaker/vdirsyncer.git"
+
+  option "without-keyring", "Build without python-keyring support"
+
+  depends_on :python
```

Comment:
In that case you might as well depend on Python 3 since vdirsyncer works better with it, no?

On 5 September 2015 08:53:47 CEST, benjaminfrank <notifications@github.com> wrote:
>> @@ -0,0 +1,91 @@
>> +class Vdirsyncer < Formula
>> +  desc "Synchronize calendars and contacts"
>> +  homepage "https://github.com/untitaker/vdirsyncer"
>> +  url
>"https://pypi.python.org/packages/source/v/vdirsyncer/vdirsyncer-0.6.0.tar.gz"
>> +  sha256
>"0d2a9677b086cfbe5fe5e7cb5e55db4c1afab62cb5dc56aeaff6e95d34bf60d5"
>> +  head "https://github.com/untitaker/vdirsyncer.git"
>> +
>> +  option "without-keyring", "Build without python-keyring support"
>> +
>> +  depends_on :python
>
>That is intentional as the system python has (some) broken ssl code
>such that you can't use https connections with vdirsyncer. at least it
>didn't work for me
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/Homebrew/homebrew/pull/41208/files#r38808419

-- 
Sent from my phone. Please excuse my brevity.